pkg/controller/buildrun/generate_taskrun.go

```diff
@@ -170,6 +170,14 @@ func GenerateTaskRun(build *buildv1alpha1.Build, buildRun *buildv1alpha1.BuildRu
 		revision = *build.Spec.Source.Revision
 	}
 
+	// update output image in build spec
```

Comment:
can you update this comment to reflect what the logic is doing? 
`update output image in build spec` seems not related to the code. It is more like "retrieve imageURL from BuildRun or Build"  . Correct me if Im wrong pls.

Reply:
Yes, you are right. I have changed it to `retrieve expected ImageURL from build or buildRun`